basic-examples/bind-constants/config.yaml

```diff
@@ -0,0 +1,8 @@
+deployers:
+  image:
+    deployer_name: podman
+log:
+  level: debug    
+logged_outputs:
+  error:
+    level: debug
```

Comment:
It would be good if these files ended in newlines.

Reply:
did it